src/index-node.ts

```diff
@@ -71,4 +71,7 @@ const fs = require("fs");
 const out = fs.createWriteStream("test.png");
 const stream = canvas.createPNGStream();
 stream.pipe(out);
-out.on("finish", () => console.log("test.png was created."));
+out.on("finish", () => {
+  // eslint-disable-next-line no-console
+  console.log("test.png was created.");
```

Comment:
`console.info()` maybe to avoid disabling.

Reply:
IMO doesn't matter. It's regular practice to disable this rule where it makes sense.

Reply:
The less we disable the better.

Reply:
I still believe having `console.info` here is preferable, than having the disable comment.